src/time/utils.es

```diff
@@ -0,0 +1,31 @@
+export const cleanValue = (value) => value.replace(/:/g, '');
```

Comment:
why not simple ':' - we only gonna have 1

Reply:
user can paste any input